### PR TITLE
Style/form icons color vars

### PR DESCRIPTION
**Pull request by adrianammt**
Hello everyone! Please review.
What I did:
-Changed the focus of the inputs for better user experience. Deleted outlines. Set default values and selected options.
-Imported react icons and defined the family to use. Imported them to my footer and added active color option.
-Added htmlFor in all labels.
-Realized Safari doesn't support step, min or max in time inputs. So I deleted those options from my form. Changed the input to hh:mm only.
-Changed the prepTime input from time to select and converted the target value to number for saving.
-Created a Colors.css file and changed all colors to the global color variables.

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -1,20 +1,30 @@
...
+        activeClassName="NavIcon--active"
+      >
+        <IoTimerOutline alt="My List icon" />
       </NavLink>
-      <NavLink to="/favourites">
-        <img className="NavIcon" src={heart} alt="My Favourites icon" />
+      <NavLink
+        to="/favourites"
+        activeClassName="NavIcon--active"
+        className={"NavIcon"}
```
Comment (hellodebs) [inline src/components/Footer.js:25]: What are the curly braces here but not set with other classes?

File: src/components/SettingsForm.css [hunk 2]
```diff
@@ -3,64 +3,74 @@
...
   padding: 0.5em 2em;
   font-weight: 600;
   font-size: 1em;
   margin-top: 1.5em;
 }
 
-.saveButton:hover {
-  background-color: #0087ac;
+.saveButton:hover,
+.saveButton:active {
```
Comment (hellodebs) [inline src/components/SettingsForm.css:53]: nice  👍

**Review by hellodebs — APPROVED** [review]
Wow, this is really well done, Adriana! Looks very good, too! 👍

**Review by namirsab — CHANGES_REQUESTED** [review]
@adrianammt well done! The feedback from the last PRs was really well implemented! 💪 
I left a few comments, please take a look

**Comment by namirsab** [comment]
Ah, and one thing, I'd give the user some feedback about the successful operation when saving a class. you can check how  @johnxgaehner did it for insipiration

File: src/components/Footer.js (hunk 1 above)
Comment (adrianammt) [reply src/components/Footer.js:25]: I was trying out things and forgot that one. Tks!

File: src/components/SettingsForm.js [hunk 3]
```diff
@@ -45,90 +53,89 @@ export default function SettingsForm() {
...
-              3
-            </option>
+          <label htmlFor="prepTime" className="input-label">
+            Prep Time
+          </label>
+          <select name="prepTime" id="prepTime" className="time-select">
+            <option>10 sec</option>
+            <option>15 sec</option>
+            <option>20 sec</option>
+            <option>30 sec</option>
```
[the remark this answers is not in the document]
Comment (adrianammt) [reply src/components/SettingsForm.js:91]: Awesome! I will change it.

File: src/index.js [hunk 4]
```diff
@@ -3,6 +3,7 @@ import ReactDOM from "react-dom";
 import "./index.css";
 import App from "./App";
 import { BrowserRouter as Router } from "react-router-dom";
+import "./Colors.css";
```
[the remark this answers is not in the document]
Comment (adrianammt) [reply src/index.js:6]: Noted.

File: src/Colors.css [hunk 5]
```diff
@@ -0,0 +1,8 @@
+:root {
+  --primary-color: #2b0080;
+  --primary-color-onAction: #76297d;
+  --accent-color: #009fcc;
+  --accent-color-onAction: #0087ac;
+  --text-onDark-color: white;
+  --text-onlight-color: var(--primary-color);
+}
```
[the remark this answers is not in the document]
Comment (adrianammt) [reply src/Colors.css:8]: I will check it out. Tks!

**Pull request merged**